File: Maquillaje.java
public class Maquillaje extends Producto {
    String tipo;
    String pigmento;

    public Maquillaje(String nombre, int numSerie, double price, String fechaImportacion, int stock, String tipo,
            String pigmento) {
        super(nombre, numSerie, price, fechaImportacion, stock);
        this.tipo = tipo;
        this.pigmento = pigmento;
    }
    
    
}

File: Maquillaje.py
from Producto import Producto

class Maquillaje(Producto):
    def __init__(self, nombre, numSerie, price, fechaImportacion, stock, tipo, pigmento):
        super().__init__(nombre, numSerie, price, fechaImportacion, stock)
        self.tipo = tipo
        self.pigmento = pigmento